spritz/spritz-android/proguard-rules.pro

```diff
@@ -0,0 +1,25 @@
+# Add project specific ProGuard rules here.
```

Comment:
Lot of leftover from the IDE default wizard. Probably good idea to clean up.

Reply:
❌ in 4bfb1654368f0becc7a283392e72483f336b173b.